### PR TITLE
Ajv8 upgrade

### PR DESCRIPTION
**Pull request by Eomm**
This PR update the `ajv` dependency from v6 to v8

The main changes are:

- ajv now does not ignore duplicated `$id` in inline schemas* => for this reason I have to add a `delete` statement. we should test this branch within Fasfity
- ajv does not include `ajv-formats` by default, I have added it

## Todos

- [x] align next branch
- [x] it must be tested within Fastify

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fast-json-stringify",
-  "version": "2.5.2",
+  "version": "2.7.8",
```
Comment (mcollina) [inline package.json:3]: why this change?

**Review by mcollina — CHANGES_REQUESTED** [review]
I think there are a lot of spurious changes here

**Comment by Eomm** [comment]
I'm must align `next` to `master` first

**Comment by Eomm** [comment]
Ready to review

Tested with Fastify next here: https://github.com/fastify/fastify/actions/runs/1166101438 all green 🟢

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**